build-lib/lib.js

```diff
@@ -67,12 +67,14 @@ module.exports = function (pathConfig, env) {
     destDir: './node-lib'
   });
 
-  if (env.production) {
-    trees.push(uglifyJavaScript(funnel(trees, {
-      getDestinationPath: function (path) {
-        return path.replace(/\.js/, '.min.js');
-      }
-    })));
+  if (env == 'production') {
+    trees = [
```

Comment:
Won't this minify all trees? Previously, we were adding additional minified trees. This looks like it replaces all trees with the min versions.

Also, use a separate const over reassigning. This is a totally different tree at this point, and calling it out as `productionTrees` or `minifiedTrees` or something makes that separate identity clear.